sailthru/sailthru_client.py

```diff
@@ -333,10 +328,10 @@ def delete_template(self, template_name):
         data = {'template': template_name}
         return self.api_delete('template', data)
 
-    def save_template(self, template, template_fields=None):
-        template_fields = template_fields or {}
-        data = template_fields.copy()
-        data['template'] = template
+    def save_template(self, template, **template_fields):
```

Comment:
In Python *args, and **kwargs is optional positional arguments, optional keyword arguments. If you don't set it, it defaults to an empty tuple and empty dictionary.